packages/datagateway-common/src/detailsPanels/isis/datafileDetailsPanel.component.test.tsx

```diff
@@ -31,22 +50,65 @@ describe('Datafile details panel component', () => {
       createTime: '2019-06-11',
       description: 'Test description',
     };
-
-    (useDatafileDetails as jest.Mock).mockReturnValue({
-      data: rowData,
+    axios.get = jest.fn().mockResolvedValue({
+      data: [rowData],
     });
   });
 
   afterEach(() => {
     jest.clearAllMocks();
   });
 
-  it('renders correctly', () => {
-    const wrapper = createWrapper();
-    expect(wrapper.find('DatafileDetailsPanel').props()).toMatchSnapshot();
+  it('should render correctly', () => {
+    const { asFragment } = renderComponent({ rowData });
+    expect(asFragment()).toMatchSnapshot();
+  });
+
+  it('should show default tab on first render', async () => {
+    renderComponent({
+      rowData,
+    });
+
+    expect(
+      await screen.findByRole('tab', { name: 'datafiles.details.label' })
+    ).toHaveAttribute('aria-selected', 'true');
   });
```

Comment:
Could be good to have this test in other files with a default tab. Currently I only see it here and in `isis/datasetDetailsPanel`

Reply:
Good shout! I think I forgot to put it in other tests.